android/titanium/src/java/org/appcelerator/titanium/TiFileProxy.java

```diff
@@ -153,6 +157,15 @@ public boolean createDirectory(@Kroll.argument(optional=true) Object arg)
 	}
 
 	@Kroll.method
+	public boolean createFile()
+	{
+		Context context = TiApplication.getInstance().getApplicationContext();
```

Comment:
Is all of this necessary? I think this is hard-coding the location of the file when it shouldn't be. Don't we already create the correct TiBaseFile up in the constructor? You should be able to just do `return tbf.createFile();` here.